src/store/trees/trees.js

```diff
@@ -50,12 +51,14 @@ export default {
         enabledFeatures: [
           /** @see \Voto5\Http\Controllers\V3TreesController::get_editTree */
         ],
+        saveCurrentlyInProgress: 0,
+        isEditable: true
       },
     }
   },
 
   getters: {
-
+    isConfigured: ({ui}) => ui.isConfigured,
```

Comment:
This is declared above as `configured: false` — Should this say `ui.configured` or the others be `isConfigured`?

Reply:
Good spot. I actually just found a bug where configure was incorrectly being called multiple times. This fixes that!